kai.yaml

```diff
@@ -54,4 +54,4 @@ anchore:
   password: $KAI_ANCHORE_PASSWORD
 #  http:
 #    insecure: true
-#    timeoutSeconds: 5
+#    timeout-seconds: 5
```

Comment:
Yo man, not sure why but I think I used camelCase on purpose. We might want to dig into why mapstructure isn't picking this up. If you already know can you just reply to this comment and let me know?

Reply:
Hey @dakaneye! I tested the code out and the mapstructure renaming works if I make the tag camelCase. 

```
	TimeoutSeconds int  `mapstructure:"timeoutSeconds"`
```

I think mapstructure is parsing it correctly. Is there another reason why this should be camelCase vs the other configuration options?

Reply:
CamelCase is the way of golang 😄